src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -207,49 +220,69 @@ const AuthenticatorButton: UISchemaElementComponent<{
             id={`${iconName}-usageDescription`}
             textAlign="start"
             sx={{
-              fontSize: '.875rem',
+              fontSize: tokens.TypographySizeSubordinate,
+              color: tokens.TypographyColorSubordinate,
               margin: 0,
-              marginBlockEnd: '6px',
-              color: 'text.secondary',
+              marginBlockStart: '6px',
             }}
             data-se="authenticator-button-usage-text"
             className="authenticator-usage-text"
           >
             {usageDescription}
           </Typography>
         )}
+        {isEnroll && (
+          <Box
+            className="cta-button authenticator-button"
+            data-se={dataSe}
+            sx={{
+              display: 'flex',
+              alignItems: 'center',
+              marginBlockStart: '14px',
+              marginInline: 0,
+              color: tokens.TypographyColorAction,
+              '& svg': {
+                marginBlock: 0,
+                marginInline: isAdditionalEnroll ? '0 5px' : '5px 0',
+              },
+            }}
+          >
+            <Box
+              component="span"
+              id={`${iconName}-ctaLabel`}
+              sx={{
+                fontWeight: tokens.TypographyWeightBodyBold,
+                fontSize: tokens.TypographySizeBody,
+                order: isAdditionalEnroll ? 1 : 0,
+              }}
+              data-se="cta-button-label"
+              className="button select-factor link-button"
+            >
+              {ctaLabel}
+            </Box>
+            {ctaIcon}
+          </Box>
+        )}
+      </Box>
+      {!isEnroll && (
```

Comment:
nit: combine with the `{isEnroll && ... }` code above and make it a ternary

Reply:
It can't be easily converted to ternary, cause the Box for enroll has [this parent](https://github.com/okta/okta-signin-widget/pull/3499/files#diff-af107efbe8a423311293438bc3e6b500a1955db3708b05954bbac1abb936e026R151) (content flex box with column direction) and the Box for verify has [another parent](https://github.com/okta/okta-signin-widget/pull/3499/files#diff-af107efbe8a423311293438bc3e6b500a1955db3708b05954bbac1abb936e026R102) (root button box)